app/src/main/AndroidManifest.xml

```diff
@@ -28,6 +28,22 @@
                 <data android:scheme="https"/>
             </intent-filter>
         </activity>
+
+        <activity android:name=".IntentReceiverActivity">
+            <intent-filter>
```

Comment:
Doesn't HomeActivity already have the VIEW, DEFAULT, and BROWSABLE flags? Should we be migrating those flags instead?